### DISCUSSION TITLE
Request: Don't spellcheck comments

### DISCUSSION THREAD

**Discussion by nivaca** [minad/jinx#132] [Q&A] [answered]
Hi! Is there a way of making Jinx skip the comments in a document (e.g. those beginning in `%` in LaTeX)? Thanks.

**Comment by minad** [comment, marked as answer]
Yes, it should be possible to ignore comments by adding `font-lock-comment-face` to `jinx-exclude-faces` for `tex-mode` as follows:

~~~ elisp
(cl-pushnew 'font-lock-comment-face (alist-get 'tex-mode jinx-exclude-faces))
~~~

Please let me know if it works for you!

**Comment by nivaca** [comment]
Great! Many thanks.

**Reply by minad** [reply]
Thanks. We can keep the question open. Maybe the information is also useful for others.
